### DISCUSSION TITLE
What's the difference between RubyGems and Gems?

### DISCUSSION THREAD

**Discussion by riotrah** [topgrade-rs/topgrade#346] [Q&A] [answered]
I see the source code and can't tell how `gem` is not related to `gem` from Ruby? idk if my phrasing even makes sense.

> Very unfamiliar with ruby in general, but have some CLI tools that I used gem to acquire, so `topgrade` invokes those steps.

**Comment by GideonBear** [comment, marked as answer]
See https://github.com/topgrade-rs/topgrade/issues/212
`gem` updates Ruby gems (`gem update`), `ruby_gem` updates the RubyGen tool itself (`gem update --system`).
This is undocumented, which should be improved (see also #761).
